keps/sig-scheduling/5598-opportunistic-batching/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: Scheduling cache
+kep-number: 5598
+authors:
+  - "@bwsalmon"
+owning-sig: sig-scheduling
+participating-sigs:
+status: provisional
+creation-date: 2025-10-01
+reviewers:
+  - "@dom4ha"
+  - "@helayoty"
+approvers:
+  - "@wojtec-t"
```

Comment:
Remove me from approvers - I'm already listed in prr file. You can add me to reviewers.